cmd/tether/attach_windows.go

```diff
@@ -29,12 +29,37 @@ import (
 
 	"github.com/vmware/vic/cmd/tether/msgs"
 	"github.com/vmware/vic/pkg/serial"
+	"github.com/vmware/vic/pkg/trace"
 )
 
-func backchannel(ctx context.Context, conn *net.Conn) error {
+const (
+	com = "COM1"
+)
 
+func rawConnectionFromSerial() (*net.Conn, error) {
+	var err error
+
+	// redirect backchannel to the serial connection
+	log.Infof("opening %s%s for backchannel", pathPrefix, com)
+	// TODO: set read timeout on port during open
+	_, err = OpenPort(fmt.Sprintf("%s%s", pathPrefix, com))
+	if err != nil {
+		detail := fmt.Sprintf("failed to open serial port for backchannel: %s", err)
+		log.Error(detail)
+		return nil, errors.New(detail)
+	}
+
+	log.Errorf("creating raw connection from %s\n", com)
+
+	// TODO: sort out the named port impl so that we can transparently switch from that to/from
+	// regular files for testing
+	// t.conn, err := serial.NewTypedConn(f, "file")
+	return nil, nil
+}
+
+func backchannel(ctx context.Context, conn *net.Conn) error {
 	// HACK: currently RawConn dosn't implement timeout so throttle the spinning
-	ticker := time.NewTicker(50 * time.Millisecond)
+	ticker := time.NewTicker(10 * time.Millisecond)
```

Comment:
I assumed that darwin code never used hence removed it not to create an illusion :)